subsys/bluetooth/audio/Kconfig.has

```diff
@@ -7,6 +7,7 @@
 menuconfig BT_HAS
 	bool "Hearing Access Service support [EXPERIMENTAL]"
 	select EXPERIMENTAL
+	select BT_SMP_SC_PAIR_ONLY
```

Comment:
Should we also add `depends on BT_SMP_MIN_ENC_KEY_SIZE == 16`, or some other way of satisfying that?

Reply:
Right!

Reply:
@Thalley To be honest I was not aware of such Kconfig option